### PR TITLE
adding helm

### PR DESCRIPTION
**Pull request by karannaoh**

### CODE REVIEW

File: charts/guillotina/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    run: {{ .Values.name }}
+spec:
+  ports:
+  - protocol: TCP
+    port: {{.Values.guillotinaPort}}
+    targetPort: {{.Values.guillotinaPort}}
+    nodePort: {{.Values.exposedPort}}
+  selector:
+    run: {{ .Values.name }}
+  type: NodePort
```
Comment (vangheem) [inline charts/guillotina/templates/service.yaml]: Make this configurable

**Comment by karannaoh** [comment]
> Also make sure that we have an easy way to customize the config.json
> 
> Otherwise, make all those settings be in values.yaml

@vangheem a person can just add a new config.json file according to his need, as we generally do while working. Along with it he as command line options too(which I guess you are asking) by passing an array of argument.

**Comment by vangheem** [comment]
@karannaoh I would really love to have a good default config that is customizatable with helm chart values.

Having to provide your own config file completely makes the helm chart much less useful.

**Comment by karannaoh** [comment]
> @karannaoh I would really love to have a good default config that is customizatable with helm chart values.
> 
> Having to provide your own config file completely makes the helm chart much less useful.

https://github.com/plone/guillotina/blob/0a02ce71a40f3408a0c63e42aa06692d734984c6/helm/guillotina/templates/configmap.yaml This is something I tried before(having config.yaml in configmap), I will define all the values inside values.yaml.

**Comment by karannaoh** [comment]
@vangheem are there more changes you want me to work on?

File: charts/guillotina/values.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+serviceType: NodePort
+
+resources: {}
+
+guillotinaPort: 8080
+
+arguments: []
+
+configurations:
```
Comment (vangheem) [inline charts/guillotina/values.yaml]: Can we rename this to `app_settings`?

Comment (karannaoh) [reply charts/guillotina/values.yaml]: yep done

**Comment by karannaoh** [comment]
@vangheem can we merge it?

**Comment by vangheem** [comment]
I think it looks good but didn't we kind of decide that we were going to just work on it here but then the actual chart would go into helm's charts?

**Comment by karannaoh** [comment]
> I think it looks good but didn't we kind of decide that we were going to just work on it here but then the actual chart would go into helm's charts?

I will create a PR to helm's charts repository. can we have one our own repository in guillotinaweb?

**Comment by vangheem** [comment]
@karannaoh hmmm, if we think we want it in both places, maybe we should just merge this then? I'm okay with merging this--just need to know what the best course of action is here.

**Comment by karannaoh** [comment]
@vangheem I tried to figure out what can be best.. I am seeing most of the projects maintain separate repositories for helm charts.

**Comment by vangheem** [comment]
Let's just merge here. It'll be easier to maintain.

**Pull request merged**